swiftui/PasswordlessSwiftUI/PasswordlessSwiftUI/ContentView.swift

```diff
@@ -154,26 +160,29 @@ struct SuccessView: View {
 }
 
 struct ContentView_Previews: PreviewProvider {
-    static var previews: some View {
-      ContentView()
-    }
+  static var previews: some View {
+    ContentView()
+  }
 }
 
 // MARK: Example Auth View Modifier
 
 extension View {
   func handlePasswordlessLogin(forEmail email: String,
```

Comment:
I would say if we're going to use this, let's keep it, otherwise let's delete it.

Reply:
I'm good if you want to use it - you may need to change the `String` argument to `Binding<String>` though (I think...)